tools/docker/Dockerfile.integ-test

```diff
@@ -31,14 +31,16 @@ RUN mkdir -p \
   /etc/cni/net.d
 
 # Install containerd to have runc shim.
-ENV CTRD_VERSION="1.6.4"
-RUN wget --quiet -O- https://github.com/containerd/containerd/releases/download/v$CTRD_VERSION/containerd-$CTRD_VERSION-linux-amd64.tar.gz | tar zxf - -C /tmp/ && \
+ENV CTRD_VERSION="1.6.8"
```

Comment:
Should this be 1.6.9?

Reply:
In the S3 bucket we have 1.6.8 version of containerd

Reply:
Can you add this to the comment on why we have 1.6.8 and a link if possible.

Reply:
Nothing specific. While adding ARM instance to the Firecracker ContainerCI, I have added the updated version 1.6.8 containerd and ctr binaries accordingly.

Reply:
On second thought, since we are only installing `/tmp/bin/containerd-shim-runc-v2`, I think it should be fine. Though we should look into upgrading CI to the same containerd version that fc-cd builds with.  Approving.